.Lib9c.Tools/Program.cs

```diff
@@ -1,11 +1,10 @@
-﻿using System;
 using Cocona;
-using Lib9c.Tools.SubCommend;
+using Lib9c.Tools.SubCommand;
 
 namespace Lib9c.Tools
 {
-    // FIXME more detailed description need
-    [HasSubCommands(typeof(Create), Description = "Create Genesis Block")]
+    [HasSubCommands(typeof(Genesis), Description = "Manage genesis block.")]
+    [HasSubCommands(typeof(Tx), Description = "Manage transactions")]
```

Comment:
```suggestion
    [HasSubCommands(typeof(Tx), Description = "Manage transactions.")]
```

In the upper line, it seems to use *dot* at the end.